frontend/src/pages/Users/Users.js

```diff
@@ -69,8 +83,29 @@ const Users = props => {
             </Fab>
           </div>
         ) : null}
-        {tabIndex === 0 && <UsersTable permissionIconDisplayed={permissionIconDisplayed} {...props} />}
+        {tabIndex === 0 && (
```

Comment:
This is difficult to read. Suggestions: Sum up conditions and change tabIndex=== 0  to something more understandbale like `isUserListShown` or `UsersTabActive`

Reply:
done